NEWS.md

```diff
@@ -1,5 +1,10 @@
 # rSOILWAT2 v6.0.2-9000
-
+* This version produces the same output as the previous version.
+* Update `SOILWAT2` to v7.1.0 which prepares for thread-safety and reentrancy
+* C-side of `rSOILWAT2` gains new globals of the four main abstract types in
+  `SOILWAT2` - SW_ALL, SW_OUTPUT_POINTERS, LOG_INFO, and PATH_INFO -
+  to interact with `SOILWAT2`.
+* Removed `OUTSEP` from `outsetup.in` within `extdata`
```

Comment:
The bullet point in the NEWS "Removed `OUTSEP` from `outsetup.in` within `extdata`" is correct but is confusing users of the R package. rSOILWAT2 never exposed users (directly) to `OUTSEP` (and didn't use it internally either). Please remove this bullet point.